### PR TITLE
[WIP] Add datatimepicker field

### PR DESCRIPTION
**Pull request by manuquentin**
- [x] Add field
- [ ] Add doc

### CODE REVIEW

**Comment by fzaninotto** [comment]
So far, so good. I'm worried about the kB this is going to add to `ng-admin.min.js`, we really need to consider #250 soon.

File: src/javascripts/ng-admin.js [hunk 1]
```diff
@@ -11,9 +11,12 @@ require.config({
         'angular-resource': 'bower_components/angular-resource/angular-resource',
         'angular-sanitize': 'bower_components/angular-sanitize/angular-sanitize',
         'angular-ui-router': 'bower_components/angular-ui-router/release/angular-ui-router',
+        'bootstrap': 'bower_components/bootstrap/dist/js/bootstrap',
```
Comment (fzaninotto) [inline src/javascripts/ng-admin.js]: seems that only bootstrap transitions and collapse are needed at this point (http://eonasdan.github.io/bootstrap-datetimepicker/Installing/).

File: src/javascripts/ng-admin/Crud/field/maDateTimeField.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    'use strict';
+
+    var datetimepicker = require('bootstrap-datetimepicker');
+
+    /**
+     * Edition field for a date with time - a text input with a datepicker.
+     *
+     * @example <ma-date-time-field field="field" value="value"></ma-date-time-field>
+     */
+    function maDateField() {
```
Comment (fzaninotto) [inline src/javascripts/ng-admin/Crud/field/maDateTimeField.js]: should probably be renamed to `maDateTimeField`

**Comment by fzaninotto** [comment]
Seing the number of added dependencies (jQuery, moment, bootstrap js), I wonder if it's not better to add https://github.com/zhaber/datetimepicker (even though it has inferior UI). Your datetimepicker, better but heavier, could be provided as an extension... Once we know how to handle them (cf #294). 

What do you think?

**Comment by manuquentin** [comment]
I've spent 3 hours trying to use [zhaber/datetimepicker](https://github.com/zhaber/datetimepicker) but apparently [it's not compatible with angular 1.3](https://github.com/zhaber/datetimepicker/issues/15).

I'll search for another directive.

**Comment by fzaninotto** [comment]
Ah, bummer. Thanks for trying.

**Comment by fzaninotto** [comment]
Any progress for documentation ?

**Comment by fzaninotto** [comment]
Superseded by #348

**Pull request closed without merging**